### PR TITLE
Fix incorrect ordering of breadcrumbs

### PR DESCRIPTION
**Pull request by tadhglewis**
Screen Shot 2022-06-15 at 1 01 13 pm

Screen Shot 2022-06-15 at 1 01 26 pm

Screen Shot 2022-06-15 at 1 01 44 pm

### CODE REVIEW

**Comment by tadhglewis** [comment]
Even with this change, the breadcrumbs are a bit weird when compared to traditional breadcrumbs.

This lists closest > broadest with closest being bold which is not like normal breadcrumbs - maybe it is fine as it's kinda more of a reverse breadcrumb to see where the closest crumb comes from

I think it'd also look weird to do broadest > closest with closest being bold

Screen Shot 2022-06-15 at 1 26 42 pm

**Comment by tadhglewis** [comment]
Maybe we should reverse the arrows but keep **closest** < broadest

Screen Shot 2022-06-15 at 1 31 18 pm

File: .changeset/young-bees-smoke.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'wingman-fe': patch
+---
+
+LocationSuggest, LocationLookup, JobCategoryLookup: fix incorrect ordering of breadcrumbs
```
Comment (72636c) [inline .changeset/young-bees-smoke.md:5]:
```suggestion
JobCategoryLookup, LocationLookup, LocationSuggest: Fix incorrect ordering of breadcrumbs
```

**Review by 72636c — APPROVED** [review]

**Comment by tadhglewis** [comment]
Done as per https://github.com/seek-oss/wingman/pull/796#issuecomment-1155944707 ... not a big fan of left arrows as they look weird but it'll do

**Pull request merged**